include/data/nodeid.h

```diff
@@ -28,19 +28,25 @@ namespace data {
 class NodeID {
   static const size_t WIDTH_ = 24;
   uint8_t value[WIDTH_];
+  std::random_device random_;
```

Comment:
This should be mt19937 initialized from random_device.

random_device is very slow and we don't need security here.